packages/integration-sdk-core/package.json

```diff
@@ -23,8 +23,9 @@
     "prepack": "yarn build:dist"
   },
   "dependencies": {
-    "@jupiterone/data-model": "^0.57.0",
+    "@jupiterone/data-model": "^0.61.3",
     "@jupiterone/integration-sdk-entity-validator": "^12.7.0",
+    "@sinclair/typebox": "^0.32.30",
```

Comment:
Can typebox be a dev dependency?

Reply:
No because we need it in runtime to generate the schema.